addons/account/tests/test_chart_template.py

```diff
@@ -38,160 +81,156 @@ def create_tax_template(cls, name, template_name, amount):
+        tax_1_new = self.env['account.tax'].search([('company_id', '=', self.company.id), ('name', '=', self.tax_template_1.name)])
         self.assertEqual(len(tax_1_old), 1, "Old tax still exists but with a different name.")
         self.assertEqual(len(tax_1_new), 1, "New tax have been created with the original name.")
+
+    def test_update_taxes_multi_company(self):
+        """
+        In a multi-company environment all companies should be correctly updated.
+        """
+        company_2 = self.env['res.company'].create({
+            'name': 'TestCompany2',
+            'country_id': self.env.ref('base.be').id,
+            'account_tax_fiscal_country_id': self.env.ref('base.be').id,
+        })
+        self.chart_template.try_loading(company=company_2)
+
+        taxes_before = self.env['account.tax'].search([('name', '=', self.tax_template_1.name)])
+        self.assertEqual(taxes_before.mapped('amount'), [self.tax_template_1.amount] * len(taxes_before))
+
+        self.tax_template_1.amount += 1
+        update_taxes_from_templates(self.env.cr, self.chart_template_xmlid)
+
+        taxes_after = self.env['account.tax'].search([('name', '=', self.tax_template_1.name)])
+        self.assertEqual(taxes_after.mapped('amount'), [self.tax_template_1.amount] * len(taxes_after))
+
+    def test_message_to_accountants(self):
+        """
+        When we duplicate a tax because it was too different from the existing one we send
+        a message to accountant advisors. This message should only be sent to advisors
+        and not to regular users.
+        """
+        # create 1 normal user, 2 accountants managers
+        accountant_manager_group = self.env.ref('account.group_account_manager')
+        advisor_users = self.env['res.users'].create([{
+            'name': 'AccountAdvisorTest1',
+            'login': 'aat1',
+            'password': 'aat1aat1',
+            'groups_id': [(4, accountant_manager_group.id)],
+        }, {
+            'name': 'AccountAdvisorTest2',
+            'login': 'aat2',
+            'password': 'aat2aat2',
+            'groups_id': [(4, accountant_manager_group.id)],
+        }])
+        normal_user = self.env['res.users'].create([{
+            'name': 'AccountUserTest1',
+            'login': 'aut1',
+            'password': 'aut1aut1',
+            'groups_id': [(4, self.env.ref('account.group_account_user').id)],
+        }])
+        # create situation where we need to recreate the tax during update to get notification(s) sent
+        self.tax_template_1.amount += 1
+        update_taxes_from_templates(self.env.cr, self.chart_template_xmlid)
+
+        # accountants received the message
+        message = self.env['mail.message'].search(
+            [('partner_ids', 'in', advisor_users.partner_id.ids), ('body', 'like', f"%{self.tax_template_1.name}%")])
```

Comment:
```suggestion
        message = self.env['mail.message'].search_count([('partner_ids', 'in', advisor_users.partner_id.ids), ('body', 'like', f"%{self.tax_template_1.name}%")])
```
You didn't indent like this anywhere else in the file. Keep it consistent.